packages/dotcom-ui-data-embed/src/client/index.ts

```diff
@@ -0,0 +1,9 @@
+import loadDataEmbed from './loadDataEmbed'
+import DataEmbedStore from './DataEmbedStore'
+
+const init = ({ id }: { id: string }) => {
```

Comment:
Not sure if it's convention, but are index files usually for just listing the modules by importing and exporting them? I.e. should this function live in its own file and be imported?

I say this when it could easily be possible that we have functions in `index` files elsewhere in this repo. Am just thinking out loud.

Reply:
This follows what `app-context` and `flags` are doing. I agree it's a bit odd though.

Reply:
Fair enough - happy to leave as is.